easypost/report.py

```diff
@@ -8,7 +8,20 @@ class Report(AllResource, CreateResource):
     def create(cls, api_key=None, **params):
         """Create a report."""
         requestor = Requestor(api_key)
-        url = "%s/%s" % (cls.class_url(), params["type"])
+        url = f"{cls.class_url()}/{params.get('type')}?"
```

Comment:
If the user doesn't specify a type, this will now populate the url with `None` and will fail at the API level, which is probably fine because previously we'd throw a `KeyError` if there was no type. 

In a follow up, we should probably add validation for reports across all libs to check if the type is present and error without a type specified.

Reply:
Sounds good, I will add an internal ticket for this.